commercetools-models/src/main/java/io/sphere/sdk/products/commands/updateactions/ChangeAssetName.java

```diff
@@ -52,10 +51,18 @@ public LocalizedString getName() {
     }
 
     public static ChangeAssetName ofVariantId(final Integer variantId, final String assetId, final LocalizedString name) {
-        return new ChangeAssetName(assetId, variantId, null, name);
+        return ofVariantId(variantId, assetId, name, true);
```

Comment:
We should use `null` instead of `true` here and do this consistently for all update actions. Our API documentation states that the default is `true`, meaning that if we pass in `null` our backend will treat it like `true`. Using `null` here makes it more consistent with our other update actions. And I think if you use the replace from idea like: 
find `, true);` replace `, null);`
and review the changes it shouldn't cause too much work. But please let me know if fixing this issue would require a lot of effort.